app/code/Magento/Sales/Block/Order/Email/Items/Order/DefaultOrder.php

```diff
@@ -96,7 +96,7 @@ public function getProductAdditionalInformationBlock()
      * @param OrderItem $item
      * @return string
      */
-    public function getItemPrice(OrderItem $item)
+    public function getItemPrice($item)
```

Comment:
I would suggest to update the PHPDOC accordingly:
`@param OrderItem|InvoiceItem|CreditmemoItem $item`

See: `\Magento\Sales\Block\Order\Email\Items\DefaultItems::getItemPrice`

Reply:
Thank you for the review, I updated the documentation